### PR TITLE
Adjusted scroll point on filtering, made results load sooner on user scroll

### PR DESCRIPTION
**Pull request by sharissis-brei**

### CODE REVIEW

File: subprojects/degrees-and-programs/source/static/app/js/degreesAndProgramsApp.js [hunk 1]
```diff
@@ -190,9 +194,9 @@ var chapman = chapman || {};
 			});
 
 			// Click on Interests in Undergraduate section
-			dap.undergraduate.$interestsItems.on('change', function () {
-				_this.mobileScrollToTarget(dap.undergraduate.$filterTypes); // Scroll to school and keyword filters on mobile
-			});
+			// dap.undergraduate.$interestsItems.on('change', function () {
+			//
+			// });
```
Comment (tatwell) [inline subprojects/degrees-and-programs/source/static/app/js/degreesAndProgramsApp.js]: Please remove dead code here and elsewhere.

**Comment by sharissis-brei** [comment]
@tatwell The commented out lines have been removed. No updates are needed to Cascade since minification removes commented lines anyway.

File: subprojects/degrees-and-programs/source/static/app/sass/templates/_degrees-and-programs.scss [hunk 2]
```diff
@@ -2,19 +2,19 @@
...
+	// 	.dap-feature {
+	// 		opacity: 1;
+	// 	}
+	//
+	// }
+	//
+	// .dap-feature {
+	// 	opacity: 0;
+	// 	transition: opacity $transition-time;
+	// }
```
Comment (tatwell) [inline subprojects/degrees-and-programs/source/static/app/sass/templates/_degrees-and-programs.scss]: Can we remove this?

Comment (sharissis-brei) [reply subprojects/degrees-and-programs/source/static/app/sass/templates/_degrees-and-programs.scss]: Sure, I'm sorry I missed this!

**Comment by sharissis-brei** [comment]
@kaylaziegler The merge conflicts have been resolved. It just required a rebuild of the CSS.

**Comment by kaylaziegler** [comment]
@sharissis-brei Is this ready to be merged? If I understood correctly, it's already been updated in production for a while and this pull request is just to make sure cascade-assets is up to date? Is that right?

**Comment by sharissis-brei** [comment]
@kaylaziegler Yes, it's ready to be merged. All changes in this branch are currently on Dev Cascade and also live.

**Pull request merged**